docs/source/libraries/openai/python.rst

```diff
@@ -0,0 +1,13 @@
+##########
+Python API
+##########
+
+********
```

Comment:
similar comment here about the length of these "decorator" characters, in here `*` need to match length of the text within